estate/models/estate_inherited_user.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class InheritedUser(models.Model):
```

Comment:
Convention is to use the same name as the name of the model (same comment for the name of the file)
```suggestion
class ResUsers(models.Model):
```